addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -3020,6 +3020,7 @@ export class OdooEditor extends EventTarget {
                     this.toolbar.style.visibility = show ? 'visible' : 'hidden';
                 }
                 if (show === false) {
+                    this.clearMenus();
```

Comment:
As a rule in the editor we prefer not to create a function if it's gonna be used only once. Also, some style changes for coherence with the editor codebase:
```suggestion
                    for (const menu of this.toolbar.querySelectorAll('.dropdown-menu.show')) {
                        menu.parentElement.querySelector('[data-bs-toggle="dropdown"]').click();
                    });
```
Will `menu.parentElement.querySelector('[data-bs-toggle="dropdown"]')` always be defined? What of `menu.parentElement`? Can we guarantee it? Because otherwise we're getting tracebacks.

Reply:
From what I see from the codebase, it seems that it's always defined , but I can add a condition to ensure it doesn't raise any error